### PR TITLE
Update to CF v4

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated the project to use the latest version of Capital Framework.

## Additions

- Added new color variables from Capital Framework

## Removals

- Removed duplicate styles that have already been ported from cf-enhancements

## Changes

- Updated the CF modules to version 4.1.0 in package.json.
- Updated all the elements to use the correct class names.

## Testing

- run `grunt less && grunt scripts`.
- run `npm start` and navigate through the site.

## Review

- @Scotchester  
- @cfarm 
- @contolini 

[Preview this PR without the whitespace changes](?w=0)

## Notes

- There are a few color variable that aren't using colors from our palette. I'm leaving them as is for now because they should really be handled in the cfpb-theme.

## Todos

- Remove any duplicate styles between this project and Capital Framework.
- Update the project to use the cfpb-theme module.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by Scotchester** [comment]
This is not a problem with your PR, but looking at the expandables page, I realize that our `:last-child { margin-bottom: 0; }` rule has eliminated the bottom spacing in all expandables, which were counting on their last child's bottom margin to be there. I'll file an issue in capital-framework.

**Comment by Scotchester** [comment]
Got a couple issues on the Tables page:

screen shot 2017-03-23 at 15 50 02

screen shot 2017-03-23 at 15 50 22

**Comment by jimmynotjim** [comment]
Updated with @Scotchester fix for pagination and markup corrections for the tables. I removed the 100% inline widths because I'd like to solve that issue in cf-tables. I don't know if we should hold this up for that or not though.

**Comment by jimmynotjim** [comment]
It's going to get messy dealing with ongoing changes to the less files with this project on v3. Can we merge this?

**Comment by Scotchester** [comment]
Testing this again locally.

> I removed the 100% inline widths because I'd like to solve that issue in cf-tables.

Are you proposing we set all table widths to 100% by default? I'm not sure we want to do that.

But anyway, a more efficient way to do it in this project would be to add a style for it in the project's stylesheets. I'll commit that, along with a fix for the pagination style to correctly show how it would look on a small screen.

We should also decide whether the expandable bottom padding issue should be a blocker to moving forward with this. As soon as we publish, those examples will look broken.

**Comment by jimmynotjim** [comment]
@Scotchester agreed that the best option would be setting them in the main stylesheet of this project. As far as the expandables, yes, we should fix that first. I can do that after I make the hero updates today.

**Comment by Scotchester** [comment]
Whoops, pushed that commit to my fork yesterday afternoon. Pushed it here now.

File: Gruntfile.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ module.exports = function(grunt) {
         tasks: ['js']
       },
       content: {
-        files: ['**/*.html', '/downloads/*.md', '/guides/*.md', '/identity/*.md', '/ui-toolkit/*.md'],
+        files: ['index.html', '**/*.md'],
```
Comment (jimmynotjim) [inline Gruntfile.js]: Why are we even converting these? Let's let Jekyll handle the markdown files, just like we did on the CF docs site.

File: src/static/css/content.less [hunk 2]
```diff
@@ -765,3 +765,32 @@ ul {
...
+
+/* Page components/Tables page
+   ========================================================================== */
+
+// Display all example tables at 100% width
+table {
+    width: 100%;
+}
+
+// Display pagination examples in half columns as their mobile view
```
Comment (jimmynotjim) [inline src/static/css/content.less]: I still need to try the iframe suggestion @niqjohnson had. Maybe I can find time on Friday.

File: Gruntfile.js (hunk 1 above)
Comment (Scotchester) [reply Gruntfile.js]: They're not being converted, just triggering the copy task. The main use case would be adding new images to a page. On saving the MD file, images will be recopied from `./src/static/img/` (everything in `src` is ignored by Jekyll) to `./static/img`, where they'll be picked up by Jekyll's watcher.

We could consider altering/getting rid of the `src`/"`dist`" model that's in use here and just keep static files that need no building in `./static`.

**Comment by jimmynotjim** [comment]
Expandables bug is being addressed at https://github.com/cfpb/AtomicComponent/pull/4 and https://github.com/cfpb/capital-framework/pull/471

**Comment by jimmynotjim** [comment]
@Scotchester I think I've addressed everything we discovered in this process. Hopefully we can merge this and keep moving on other improvements.

File: Gemfile.lock [hunk 3]
```diff
@@ -0,0 +1,200 @@
+GEM
```
Comment (Scotchester) [inline Gemfile.lock]: How about we go ahead and remove & ignore `Gemfile.lock`?

File: package.json [hunk 4]
```diff
@@ -20,16 +20,16 @@
...
+    "cf-buttons": "^4.1.0",
+    "cf-core": "^4.1.0",
+    "cf-expandables": "^4.1.0",
+    "cf-forms": "^4.1.0",
+    "cf-grid": "^4.1.0",
+    "cf-icons": "^4.1.0",
+    "cf-layout": "^4.1.0",
+    "cf-pagination": "^4.1.0",
+    "cf-tables": "^4.1.0",
+    "cf-typography": "^4.1.0",
```
Comment (Scotchester) [inline package.json:32]: Several of these need to get updated to their latest releases. Easiest thing to reference is the badges at the top of the capital-framework README.

**Review by Scotchester — CHANGES_REQUESTED** [review]
Need a quick update to the `package.json` file to make sure I'm testing things totally cleanly, rather than relying on npm links.

File: package.json (hunk 4 above)
Comment (jimmynotjim) [reply package.json:32]: @Scotchester the `^` takes care of that for you, run `npm update`.

File: Gemfile.lock (hunk 3 above)
Comment (jimmynotjim) [reply Gemfile.lock]: That will go away when I rebase, @marteki removed that already.

**Review by Scotchester — APPROVED** [review]

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**